app/user/rest.py

```diff
@@ -792,7 +791,7 @@ def _update_alert(
         notification_type=template.template_type,
         api_key_id=None,
         key_type=KEY_TYPE_NORMAL,
-        reply_to_text=reply_to,
+        reply_to_text=None,
```

Comment:
The default is None, no need for this. Applies to all `persist_notification` calls in this file.

Reply:
deleted